src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -12,13 +15,22 @@ public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_SUCCESS = "Listed all transactions";
+
+    private static final Logger logger = Logger.getLogger("ListCommandLogger");
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all transactions in UniCash. "
```

Comment:
minor aesthetic consideration/nit, do we wanna standardize all non-functional String mentions of UniCash to UniCa$h? there wouldn't be any issues with any validation regex right